server/store/migrations/000014_survey_table_add_team_filter_type_column.up.sql

```diff
@@ -0,0 +1,7 @@
+{{- /* you annot add a new colum with not null constraint without a default value */ -}}
+{{- /* so, we're adding the column and setting the default value we need to set in existing rows */ -}}
+{{- /* then removing the default value from column as its no longer needed */ -}}
```

Comment:
This style of comment writing is a bit odd to me. Could we not just write this as a regular SQL comment like this? Also, there's a couple letters missing in the first line
```suggestion
-- you cannot add a new column with not null constraint without a default value
-- so, we're adding the column and setting the default value we need to set in existing rows
-- then removing the default value from column as its no longer needed
```

Reply:
Done.